app/src/components/common/form/dropdown/index.tsx

```diff
@@ -83,7 +83,7 @@ const DropdownVariantCardCSS = css`
   flex: 1;
   padding: 13px 25px;
   position: relative;
-  border: 1px solid #dcdff2;
+  border: 1px solid ${props => props.theme.colors.tertiary};
```

Comment:
Though it's the same color value, this should be `props.theme.dropdown.buttonBorderColor`